browser/ui/webui/padded_image_source_unittest.cc

```diff
@@ -0,0 +1,3643 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/ui/webui/padded_image_source.h"
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "base/containers/span.h"
+#include "base/memory/ref_counted_memory.h"
+#include "base/test/bind.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/browser/ui/webui/sanitized_image_source.h"
+#include "chrome/test/base/testing_profile.h"
+#include "content/public/browser/web_contents.h"
+#include "content/public/test/browser_task_environment.h"
+#include "services/data_decoder/public/cpp/test_support/in_process_data_decoder.h"
+#include "services/network/public/cpp/simple_url_loader.h"
+#include "services/network/public/cpp/weak_wrapper_shared_url_loader_factory.h"
+#include "services/network/test/test_url_loader_factory.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+namespace {
+
+constexpr unsigned char kValidImage[] = {
```

Comment:
![image](https://github.com/user-attachments/assets/cc8edcfd-512d-45fd-8264-d3135c39ea8e)

just store it as a test file and read into a variable instead of adding 3k source lines.

Reply:
![img-monocle](https://github.com/user-attachments/assets/7b113cbc-7775-4b8c-abb1-42847fcae2f6)